### DISCUSSION TITLE
Switching from nerdctl to dockerd/moby error

### DISCUSSION THREAD

**Discussion by Exagram** [rancher-sandbox/rancher-desktop#1800] [Q&A] [answered]
Hi Rancher Desktop,

Is [this](https://github.com/rancher-sandbox/rancher-desktop/issues/1801) a bug or user mistake?

Steps:
- Install Rancher Desktop on Windows 10
- Kubernetes Settings > `containerd`
- WSL Integration > Enable Windows WSL in "Ubuntu"
- Realize nerdctl isn't compatible with `skaffold`, switch to `dockerd/moby`
- In WSL bash run either `docker ps` or `docker images`

Expected:
- A list of hashes

Actual:
```sh
FATA[0000] cannot access containerd socket "/run/k3s/containerd/containerd.sock" (hint: try running with `--address /var/run/docker/containerd/containerd.sock` to connect to Docker-managed containerd): no such file or directory
```

Discussion:
- Strange. Did I do something wrong? If not I'll file an issue.

Answer:
- If you ran `echo alias docker=nerdctl >> ~/.profile` make sure to delete this line and restart WSL via `wsl -t ubuntu`!

**Comment by jandubois** [comment, marked as answer]
> Answer:
> 
> * If you ran `echo alias docker=nerdctl >> ~/.profile` make sure to delete this line and restart WSL via `wsl -t ubuntu`!

I don't quite understand the question; you seem to have to include the answer in your post as well. Are you just documenting what went wrong for you, or are you waiting for some additional answers?

**Reply by Exagram** [reply]
I have solved my problem and provided the answer in the same post.

(Off-topic: Strange. Why am I unable to delete posts?)

**Reply by ajkret** [reply]
It is not an answer, it is a workaround. I have a similar problem, Rancher Desktop on Windows does not work with moby / dockerd.
